recipes/doctest/2.3.5/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+project(test_package)
+cmake_minimum_required(VERSION 3.3)
+
+set(CMAKE_VERBOSE_MAKEFILE TRUE)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+file(GLOB SOURCE_FILES *.cpp)
+option(TEST_PACKAGE_DEFINE_MAIN "" ON)
+
+add_executable(${PROJECT_NAME} ${SOURCE_FILES})
+if (TEST_PACKAGE_DEFINE_MAIN)
+  target_compile_definitions(${PROJECT_NAME} PRIVATE DEFINE_MAIN=1)
```

Comment:
Why don't you export this define directly from `package_info` ?

Reply:
oops

Reply:
Actually, not oops, there is no flag defined by doctest to indicate that main is inside the static lib. So I wonder if it's the recipe's role to do it.